me.stonegate.tsacdop.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
has this been submitted upstream?

Reply:
I can confirm that I am the developer.

Reply:
should be maintained upstream for future releases

Reply:
You are right, I will fix this, and also desktop file, right?

Reply:
also. Neither are specific to flatpak. The .desktop is actually a requirement to have the app appear in the various desktop environments.

(it's ok for now to have them here)